tests/metagen/typegraphs/identities/ts/fdk.ts

```
@@ -33,6 +33,1181 @@ export type Handler<In, Out> = (
+  static hostcall(
+    qg: _QueryGraphBase,
+    tg: Deployment,
+    options?: Pick<GraphQlTransportOptions, "headers">,
+  ) {
+    return new HostcallTransport(
+      (doc, vars) => tg.gql([doc]).run(vars),
+      options ?? {},
+      qg.typeNameMapGql,
+    );
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)